### ISSUE TITLE
Bottom pane with terminal sessions can obscure window when sessions closed.

### ISSUE THREAD

**Issue by GrahamDumpleton** [vmware-tanzu/octant#740]
**What steps did you take and what happened:**

This one can be a bit fiddly to reproduce, but seems to happen more often than not.

Create a terminal by executing bash command in a container. This will cause terminal window pane to push up from the bottom of the page.

Resize this pane by dragging it upwards.

Now click `x` to kill the terminal session.

The pane should withdraw.

Use the arrow to expand the terminal pane again when no active terminal session. It will expand and collapse again, but sometimes when you do this it doesn't collapse the whole way, but is left overlapping the bottom of the page to the extent you resized it previously.

![image](https://user-images.githubusercontent.com/507637/75500794-15d6ad80-5a22-11ea-9bac-86770224128d.png)

**What did you expect to happen:**

Should collapse back to original smaller size.

**Anything else you would like to add:**

Nope.

**Environment:**

- Octant version (use `octant version`): 

```
$ octant version
Version:  0.10.2
Git commit:  438ce3d52088c1e3e15aaf9817ba71e3c85255f5
Built:  2020-02-15T02:33:23Z
```

- Kubernetes version (use `kubectl version`):

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-13T18:08:14Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-11T18:07:13Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
```

- OS (macOS 10.15, Windows 10, Ubuntu 19.10 etc):

```
$ uname -a
Darwin grumpy-at-home.fritz.box 19.3.0 Darwin Kernel Version 19.3.0: Thu Jan  9 20:58:23 PST 2020; root:xnu-6153.81.5~1/RELEASE_X86_64 x86_64
```

Also affects separate web browser when accessing Octant as web service.

**Comment by wwitzel3** [comment]
This will be fixed when we do #664

**Comment by mklanjsek** [comment]
This is not an issue anymore after Terminal is moved to its own tab.
